### ISSUE TITLE
Support more VIRTIO Devices

### ISSUE THREAD

**Issue by BenNottelling** [copy/v86#405]
With v86 starting to boot newer OS's, I think it would be really neat to see it supporting more hardware in the form of VIRTIO devices as these are pretty easy to find drivers for across the board.

The biggest in my opinion would be a netkvm virtio networking adapter. You can get windows drivers for them as far back as XP or as new as W10, and ReactOS ships with a driver for it as well. L:inux also has great support for it as well.

**Comment by ChrDur1** [comment]
not an admin, but I'm not sure about how to solve this

**Comment by wtzhang23** [comment]
Hello, I'm a student at UT Austin looking for open source projects related to virtualization for my final class project. Could I try tackling this?

**Comment by BenNottelling** [comment]
Go right ahead!

**Comment by hello-smile6** [comment]
https://github.com/torvalds/linux/blob/018ab4fabddd94f1c96f3b59e180691b9e88d5d8/include/uapi/linux/virtio_ids.h#L76-L82
That might help.

**Comment by hello-smile6** [comment]
https://bellard.org/jslinux/vm.html?cpu=riscv64&url=buildroot-riscv64-xwin.cfg&graphic=1&mem=256 somehow has a tablet device implemented.

**Comment by hello-smile6** [comment]
Why does jslinux emulate a VMWare mouse?

**Comment by broten15** [comment]
Hi, I'm also a student at UT Austin looking for projects related to virtualization for a class project. Is this issue up for grabs?

**Comment by wtzhang23** [comment]
Yes; feel free to take this up. I didn't have the bandwidth to solve this issue.

**Comment by copy** [comment]
> Hi, I'm also a student at UT Austin looking for projects related to virtualization for a class project. Is this issue up for grabs?

Sure! I'd suggest virtio-tablet or virtio-net. This is a fairly complex task, feel free to ping me on [our chat](https://app.gitter.im/#/room/#v86:libera.chat) for pointers.
